content/post/yet-another-reason-to-privatize-the-police.md

```diff
@@ -57,11 +74,22 @@ In addition to qualified immunity, all government-run businesses and government-
 
 The very simple metrics of profit and loss determine how successful business are in meeting the needs of consumers. Private enterprises that waste resources, drive away customers with poor service, or refuse to tailor their products and services to meet consumer demand will quickly find themselves losing revenue as consumers decide to patronize the more profitable, i.e. successful, businesses.
 
-Police departments, being financially secured through taxation and, as a last resort the printing press, face no financial consequences for squandering resources, failing to cut costs, refusing to innovate, or wrongfully killing one of their constituents. This is because the continued operation of police departments is not contingent upon the voluntary patronage of its citizens. No amount of consumer dissatisfaction, resource waste, or business malpractice will lead to a loss of revenue that would force departments to rethink their allocation of resources or business practices.
+Police departments, being financially secured through taxation and, as a last resort, the printing press face no financial consequences for squandering resources, failing to cut costs, refusing to innovate, or wrongfully killing one of their constituents. This is because the continued operation of police departments is not contingent upon the voluntary patronage of its citizens. No amount of consumer dissatisfaction, resource waste, or business malpractice will lead to a loss of revenue that would force departments to rethink their allocation of resources or business practices.
+
+## There was no reason for George Floyd to die
+
+One can only wonder whether George Floyd would have died if his arrest had been carried out by a privatized police force that was held accountable to the profit and loss system, in constant competition with other agencies to provide the most just and peaceful means of dispute resolution, and unanointed with the doctrine of qualified immunity.
+
+We live in the world's most technologically advanced and competitive economy. Normal human beings will always demand protection for themselves, their families, their property, and their businesses. In light of all the protesting, looting and overwhelming dissatisfaction with state-funded police, it is time for the free market to begin protecting and serving our communities.
+
+[^1]: If we are going to be nabbing citizens for using counterfeit money, we might want to consider proportional fines for the Federal Reserve, an institution that has been counterfeiting to the tune of trillions for more than a century.
+
+[^2]: The claim that brutality is a well-known feature of the police system is hardly irrefutable, as the term "police brutality" is the only one of its kind. Can you remember the last time you heard someone speak of "Walmart brutality," "Costco brutality," "Gamestop brutality," "Papa Johns brutality," and so on?
```

Comment:
You may want to research some of the bigger private security/protection firms to see if "brutality" comes up in reference to them.

Reply:
A cursory glance didn't produce anything meaningful